src/apps/frontend/contexts/auth.provider.tsx

```diff
@@ -18,6 +18,14 @@ type AuthContextType = {
   loginError: AsyncError;
   loginResult: AccessToken;
   logout: () => void;
+  sendOTP: (
+    countryCode: string,
+    phoneNumber: string,
+  ) => Promise<AuthRecord>;
+  verifyOTP: (
+    authRecordId: string,
+    otp: string,
+  ) => Promise<AuthRecord>;
```

Comment:
Just like login, system will send access token back. The return type is incorrect.

See this for more details: https://github.com/jalantechnologies/boilerplate-mern/pull/115#discussion_r1517143479